### ISSUE TITLE
尝试使用SPU进行一个神经网络训练，但是在运行开始报错

### ISSUE THREAD

**Issue by TY-cc** [secretflow/secretflow#1433]
### Issue Type

Documentation Feature Request

### Source

binary

### Secretflow Version

secretflow     1.8.0b0

### OS Platform and Distribution

CentOS Linux release 7.9.2009 (Core)

### Python version

Python 3.10.4

### GCC/Compiler version

gcc (GCC) 4.8.5 20150623 (Red Hat 4.8.5-44)

### What happend and What you expected to happen.

```shell
我按照这个网址https://secretflow.readthedocs.io/zh-cn/stable/tutorial/nn_with_spu.html，想试试使用spu进行神经网络训练，但是运行的提示报错这个
self.pid = _posixsubprocess.fork_exec(
2024-08-16 10:31:21,775 INFO worker.py:1724 -- Started a local Ray instance.
INFO:jax._src.xla_bridge:Unable to initialize backend 'cuda':
INFO:jax._src.xla_bridge:Unable to initialize backend 'rocm': module 'jaxlib.xla_extension' has no attribute 'GpuAllocatorConfig'
INFO:jax._src.xla_bridge:Unable to initialize backend 'tpu': INTERNAL: Failed to open libtpu.so: libtpu.so: cannot open shared object file: No such file or directory
(_run pid=22709) INFO:jax._src.xla_bridge:Unable to initialize backend 'cuda':
(_run pid=22709) INFO:jax._src.xla_bridge:Unable to initialize backend 'rocm': module 'jaxlib.xla_extension' has no attribute 'GpuAllocatorConfig'
(_run pid=22709) INFO:jax._src.xla_bridge:Unable to initialize backend 'tpu': INTERNAL: Failed to open libtpu.so: libtpu.so: cannot open shared object file: No such file or directory
(_run pid=22702) INFO:jax._src.xla_bridge:Unable to initialize backend 'cuda':
(_run pid=22702) INFO:jax._src.xla_bridge:Unable to initialize backend 'rocm': module 'jaxlib.xla_extension' has no attribute 'GpuAllocatorConfig'
(_run pid=22702) INFO:jax._src.xla_bridge:Unable to initialize backend 'tpu': INTERNAL: Failed to open libtpu.so: libtpu.so: cannot open shared object file: No such file or directory
```

### Reproduction code to reproduce the issue.

```shell
代码是该网址的神经网络训练
```

**Comment by wangzul** [comment]
可以提供一下当前的python环境吗？执行pip list

**Comment by TY-cc** [comment]
感谢回复~，环境是下面的
(sf) [root@localhost secretflow]# pip list
Package                      Version
---------------------------- ---------------
absl-py                      2.1.0
aiobotocore                  2.13.2
aiohappyeyeballs             2.3.5
aiohttp                      3.10.3
aioitertools                 0.11.0
aiosignal                    1.3.1
astunparse                   1.6.3
async-timeout                4.0.3
attrs                        24.2.0
botocore                     1.34.131
cachetools                   5.4.0
certifi                      2024.7.4
cffi                         1.17.0
charset-normalizer           3.3.2
chex                         0.1.86
clean-text                   0.6.0
click                        8.1.7
cloudpickle                  3.0.0
colorama                     0.4.6
commonmark                   0.9.1
contourpy                    1.2.1
cryptography                 43.0.0
cycler                       0.12.1
dill                         0.3.8
emoji                        1.7.0
etils                        1.7.0
filelock                     3.15.4
flatbuffers                  2.0.7
flax                         0.8.4
fonttools                    4.53.1
frozenlist                   1.4.1
fsspec                       2024.2.0
ftfy                         6.2.3
gast                         0.4.0
google-auth                  2.33.0
google-auth-oauthlib         1.0.0
google-pasta                 0.2.0
grpcio                       1.56.2
h5py                         3.11.0
idna                         3.7
importlib_resources          6.4.2
interconnection              0.2.0.dev0
jax                          0.4.26
jaxlib                       0.4.26
Jinja2                       3.1.4
jmespath                     1.0.1
joblib                       1.4.2
jsonschema                   4.23.0
jsonschema-specifications    2023.12.1
keras                        2.12.0
kiwisolver                   1.4.5
kuscia                       0.0.2b0
libclang                     18.1.1
lightning-utilities          0.11.6
llvmlite                     0.40.1
Markdown                     3.6
MarkupSafe                   2.1.5
matplotlib                   3.9.2
ml-dtypes                    0.4.0
mpmath                       1.3.0
msgpack                      1.0.8
multidict                    6.0.5
multiprocess                 0.70.16
nest-asyncio                 1.6.0
networkx                     2.8.8
numba                        0.57.0
numpy                        1.24.3
nvidia-cublas-cu12           12.1.3.1
nvidia-cuda-cupti-cu12       12.1.105
nvidia-cuda-nvrtc-cu12       12.1.105
nvidia-cuda-runtime-cu12     12.1.105
nvidia-cudnn-cu12            8.9.2.26
nvidia-cufft-cu12            11.0.2.54
nvidia-curand-cu12           10.3.2.106
nvidia-cusolver-cu12         11.4.5.107
nvidia-cusparse-cu12         12.1.0.106
nvidia-nccl-cu12             2.18.1
nvidia-nvjitlink-cu12        12.6.20
nvidia-nvtx-cu12             12.1.105
oauthlib                     3.2.2
onnx                         1.16.0
opt-einsum                   3.3.0
optax                        0.1.8
orbax                        0.1.9
orbax-checkpoint             0.5.16
packaging                    24.1
pandas                       1.5.3
pillow                       10.4.0
pip                          24.0
protobuf                     4.25.4
pyarrow                      14.0.2
pyasn1                       0.6.0
pyasn1_modules               0.4.0
pycparser                    2.22
Pygments                     2.18.0
pyparsing                    3.1.2
python-dateutil              2.9.0.post0
pytz                         2024.1
PyYAML                       6.0.2
ray                          2.9.3
referencing                  0.35.1
requests                     2.32.3
requests-oauthlib            2.0.0
rich                         11.2.0
rpds-py                      0.20.0
rsa                          4.9
s3fs                         2024.2.0
scikit-learn                 1.3.2
scipy                        1.14.0
sdc-apis                     0.1.0.dev240320
sdc-sdk                      0.1.0.dev240320
secretflow                   1.8.0b0
secretflow-rayfed            0.2.1a2
secretflow-serving-lib       0.5.0b0
setuptools                   69.5.1
sf-heu                       0.5.0b0
six                          1.16.0
sparse                       0.15.4
spu                          0.9.1b0
sympy                        1.13.2
tabulate                     0.9.0
tensorboard                  2.12.3
tensorboard-data-server      0.7.2
tensorflow                   2.12.1
tensorflow-estimator         2.12.0
tensorflow-io-gcs-filesystem 0.37.1
tensorstore                  0.1.64
termcolor                    2.4.0
tf2onnx                      1.14.0
threadpoolctl                3.5.0
toolz                        0.12.1
torch                        2.1.2
torchaudio                   2.1.2
torchmetrics                 1.3.2
torchvision                  0.16.2
tpu                          0.1.0
tqdm                         4.66.5
triton                       2.1.0
typing_extensions            4.5.0
urllib3                      2.2.2
validator.py                 1.3.0
wcwidth                      0.2.13
Werkzeug                     3.0.3
wheel                        0.43.0
wrapt                        1.14.1
xgboost                      1.7.5
yarl                         1.9.4
zipp                         3.20.0
